src/DataProviders/SqlDataProvider/SqlDataProvider.ts

```diff
@@ -0,0 +1,68 @@
+﻿//import sql from "mssql";
+const sql = require('mssql');
+import * as assert from "assert";
+import { IDataProvider } from "../IDataProvider";
+import { ConnectionPool, ConnectionError, config, IResult } from "mssql";
+import { SeasonInfo } from "../../DataModels/SeasonInfo";
+
+/*
+ * Sql Data Provider
+ */
+export class SqlDataProvider implements IDataProvider {
+    private _tablePrefix: string = "Hymns_";
+
+    constructor(private _config: config) {
+    }
+
+    private _getQualifiedName(sp: string): string {
+        return `${this._tablePrefix}${sp}`
+    }
+
+    //TODO: why is the return type requiring void?
+    public getSeasonList(): Promise<void | SeasonInfo[]> {
+        const cp = new ConnectionPool(this._config);
+        return cp.connect()
+            .then((pool) => {
+                return pool.request()
+                    .query("select * from Hymns_Seasons");
```

Comment:
Can we put the sql queries in a resources .dll?

Reply:
These will be replaced with stored procs.  Tracking issue #7